Contributing-code-to-Oppia.md

```diff
@@ -184,8 +185,7 @@ If an issue hasn't got someone assigned to it, and there's no existing PR for th
 
   ```console
   git blame -L 40,60 file-name
-  ```
-
+  
```

Comment:
Why are you changing this? It breaks the page.

Reply:
Ah, don't know how it happened...corrected it now